mixins/demo/visibility-test-item.js

```diff
@@ -0,0 +1,46 @@
+import '../../components/button/button-icon.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { VisibilityMixin } from '../visibility-mixin.js';
+
+export class VisibilityTestItem extends VisibilityMixin(LitElement) {
+
+	static get properties() {
+		return {
+			number: { type: Number, reflect: true }
+		};
+	}
+
+	static get styles() {
+		return css`
+			:host {
+				align-items: center;
+				border-bottom: 1px solid #cccccc;
```

Comment:
Looks like this is also causing a jump, probably because it's not included in scrollHeight. I'll need to modify the mixin to also include borderTop/Bottom.